index.bs

```diff
@@ -1150,17 +1152,21 @@ to render as though it was not stopped.
 
 # Referencing a SIMID creative from VAST # {#api-vast}
 
-When a SIMID creative is referenced from a VAST document, the value for the
-`apiFramework` attribute in the `InteractiveCreativeFile` element must be
-`SIMID` (all caps). This attribute identifies the SIMID API for the creative.
-Version negotiation should be handled by the [[#api-ad-loading]] algorithm
-(rather than identified in the VAST file).
-
-Another attribute of the `InteractiveCreativeFile` is `variableDuration`.
-If `variableDuration` is true this means that the ad is only playable if
-the player allows the ad to pause and extend duration. If the player does
-not support this capability, then it must not render the video or creative.
-The player should error out instead.
```

Comment:
Editor's Note: Is the 2nd-to-last sentence above *really* something we all agree on and are mandating for SIMID compliance from a media player? `...If the player does
not support this capability, then it must not render the video or creative.` ? Assuming yes, I added bold to "**must not**" in my edit below. I'm calling it out because this feels to me like something we should maybe draw even more attention to, since it's a significant business rule/restriction, and I'm not sure how we enforce compliance or tracking of that. Thoughts?